src/message.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import debug from 'debug'
+import kebabCase from 'lodash/kebabCase'
+import {sprintf} from 'sprintf-js'
+
+const dbg = debug('woonerf:message')
+
+let messages = {}
+if (process.env.MESSAGES) {
+  setMessages(JSON.parse(process.env.MESSAGES))
+}
+
+export default function getMessage (defaultMessage: string, slug: void | string, ...args): string {
+  const key = slug || kebabCase(defaultMessage)
+  const msg = messages[key] || defaultMessage
```

Comment:
A lot of our messages are nested JSON objects.  Maybe use [lodash.get](https://lodash.com/docs/4.17.4#get)?